index.js

```diff
@@ -142,16 +141,34 @@ function replaceEnvVars(config) {
     });
 }
 
+function requiresConfig(options) {
+    return !options.generate && (options.start || options.test);
```

Comment:
I must be missing something. You cannot both specify a command that needs the config and one that doesn't (as only one is allowed). It follows that neither `generate` nor `build` should be taken into account here, as they don't need it.